c7n/schema.py

```diff
@@ -42,8 +42,9 @@
 def validate(data, schema=None):
     if schema is None:
         schema = generate()
-        Validator.check_schema(schema)
-    validator = Validator(schema)
+
+    validator = Validator(schema, format_checker=FormatChecker())
```

Comment:
passing the formatchecker explicit seems to be superflous, ie. the implicit conversion is happening without that.

Reply:
actually it looks like passing in the format checker is required to validate formats: http://python-jsonschema.readthedocs.io/en/latest/validate/#validating-formats

> By default, no validation is enforced, but optionally, validation can be enabled by hooking in a format-checking object into an IValidator.